test/core/channel/metrics_test.cc

```diff
@@ -301,194 +301,160 @@ class FakeStatsPluginBuilder {
-        "optional_label_key_1", "optional_label_key_2"};
-    uint64_histogram_handle_ =
-        GlobalInstrumentsRegistry::RegisterUInt64Histogram(
-            "uint64_histogram", "A simple uint64 histogram.", "unit",
-            kLabelKeysContainer, kOptionalLabelKeysContainer);
-    double_histogram_handle_ =
-        GlobalInstrumentsRegistry::RegisterDoubleHistogram(
-            "double_histogram", "A simple double histogram.", "unit",
-            kLabelKeysContainer, kOptionalLabelKeysContainer);
+    GlobalInstrumentsRegistry::TestOnlyResetGlobalInstrumentsRegistry();
+    GlobalStatsPluginRegistry::TestOnlyResetGlobalStatsPluginRegistry();
   }
-
- protected:
-  std::shared_ptr<FakeStatsPlugin> plugin1_;
-  std::shared_ptr<FakeStatsPlugin> plugin2_;
-  std::shared_ptr<FakeStatsPlugin> plugin3_;
-
-  static GlobalInstrumentsRegistry::GlobalUInt64CounterHandle
-      uint64_counter_handle_;
-  static GlobalInstrumentsRegistry::GlobalDoubleCounterHandle
-      double_counter_handle_;
-  static GlobalInstrumentsRegistry::GlobalUInt64HistogramHandle
-      uint64_histogram_handle_;
-  static GlobalInstrumentsRegistry::GlobalDoubleHistogramHandle
-      double_histogram_handle_;
 };
 
-GlobalInstrumentsRegistry::GlobalUInt64CounterHandle
-    MetricsTest::uint64_counter_handle_;
-GlobalInstrumentsRegistry::GlobalDoubleCounterHandle
-    MetricsTest::double_counter_handle_;
-GlobalInstrumentsRegistry::GlobalUInt64HistogramHandle
-    MetricsTest::uint64_histogram_handle_;
-GlobalInstrumentsRegistry::GlobalDoubleHistogramHandle
-    MetricsTest::double_histogram_handle_;
-
 TEST_F(MetricsTest, UInt64Counter) {
+  auto uint64_counter_handle = GlobalInstrumentsRegistry::RegisterUInt64Counter(
+      "uint64_counter", "A simple uint64 counter.", "unit",
+      {"label_key_1", "label_key_2"},
+      {"optional_label_key_1", "optional_label_key_2"});
   constexpr absl::string_view kLabelValues[] = {"label_value_1",
                                                 "label_value_2"};
   constexpr absl::string_view kOptionalLabelValues[] = {
       "optional_label_value_1", "optional_label_value_2"};
-  GlobalStatsPluginRegistry::GetStatsPluginsForChannel(
-      {.target = "domain1.domain2.domain3.domain4"})
-      .AddCounter(uint64_counter_handle_, 1, kLabelValues,
-                  kOptionalLabelValues);
-  GlobalStatsPluginRegistry::GetStatsPluginsForChannel(
-      {.target = "domain2.domain3.domain4"})
-      .AddCounter(uint64_counter_handle_, 2, kLabelValues,
-                  kOptionalLabelValues);
-  GlobalStatsPluginRegistry::GetStatsPluginsForChannel(
-      {.target = "domain3.domain4"})
-      .AddCounter(uint64_counter_handle_, 3, kLabelValues,
-                  kOptionalLabelValues);
-  EXPECT_THAT(plugin1_->GetCounterValue(uint64_counter_handle_, kLabelValues,
-                                        kOptionalLabelValues),
+  constexpr absl::string_view domain1_to_4 = "domain1.domain2.domain3.domain4";
```

Comment:
These should have constant-style names, like `kDomain1To4`.

https://google.github.io/styleguide/cppguide.html#Constant_Names

Same for all tests.

Reply:
Done.